pkgs/development/tools/yarn-berry/default.nix

```diff
@@ -1,4 +1,12 @@
-{ fetchFromGitHub, lib, nodejs, stdenv, testers, yarn, berryVersion ? 4 }:
+{
+  fetchFromGitHub,
+  lib,
+  nodejs,
+  stdenv,
+  testers,
+  yarn,
+  berryVersion ? 4,
+}:
```

Comment:
It'd be a bit nicer if the formatting change would come before the `yarn-berry` 3 commit. Also, the commit title should be: `yarn-berry3: init at 3.8.6`.

Reply:
done